packages/components/src/atoms/Badge/Badge.tsx

```diff
@@ -0,0 +1,66 @@
+import type { ReactNode, HTMLAttributes, AriaAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export type BadgeVariants =
+  | 'info'
+  | 'highlighted'
+  | 'success'
+  | 'neutral'
+  | 'warning'
+  | 'danger'
+
+export interface BadgeProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+  /**
+   * Sets the component's size.
+   */
+  size?: 'small' | 'big'
+  /**
+   * Specifies the component variant.
+   */
+  variant?: BadgeVariants
+  /**
+   * Enables counter badge.
+   */
+  counter?: boolean
+  /**
+   * For accessibility purposes, defines an ARIA label to the element when counter badge
+   */
```

Comment:
I didn't understand this bit. The `aria-label` would only be added when `counter === true`? If so, then how about

```suggestion
  /**
   * For accessibility purposes, adds an ARIA label to the element when `counter` is set to `true`.
   */
```

Reply:
Yah! That's sounds a lot better. Thanks!